src/components/pages/ChatPage/ChatPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { usePubNub } from 'pubnub-react';
+import { LeftSideChat, RightSideChat } from '../../organisms';
+import { StyledContainer } from './ChatPage.styles';
+
+const ChatPage = () => {
+  const pubnub = usePubNub();
+  const userId = localStorage.getItem('userId');
+  const [channelId, setChannelId] = React.useState<string | null>(null);
+
+  const handleIdChange = React.useCallback((newChannelId: string | null) => {
+    setChannelId(newChannelId);
+  }, []);
+
+  useEffect(() => {
+    if (userId !== pubnub.getUUID()) {
+      pubnub.setUUID(userId as string);
```

Comment:
why do you use `as` operator? I think it better to assign a type to `userId` above